AUTHORS

```diff
@@ -0,0 +1,5 @@
+Sahil Tikale <sahil.tikale@gmail.com>
```

Comment:
I would vote for getting rid of the `AUTHORS` file. This is what the git commit log is for. It's too easy for an `AUTHORS` file to fall out of sync as other people work on the project.

Reply:
done